crates/pallet-domains/src/lib.rs

```diff
@@ -285,17 +287,18 @@ mod pallet {
     #[pallet::genesis_config]
     #[derive(Default)]
     pub struct GenesisConfig {
-        pub genesis_domain_runtime: Option<GenesisDomainRuntime>,
+        pub genesis_domain_runtime: Option<(GenesisDomainRuntime, RuntimeVersion)>,
     }
 
     #[pallet::genesis_build]
     impl<T: Config> GenesisBuild<T> for GenesisConfig {
         fn build(&self) {
-            if let Some(genesis_domain_runtime) = &self.genesis_domain_runtime {
+            if let Some((genesis_domain_runtime, runtime_version)) = &self.genesis_domain_runtime {
```

Comment:
why tuple here though? adding this to `genesis_domain_runtime` seems more sense to me